packages/design-system/src/styles/components/_Table.scss

```diff
@@ -1,13 +1,18 @@
 @import '../settings/index.scss';
 
+$table-header-background-color: $color-gray-lightest !default;
+$table-stripe-background-color: $color-gray-lightest !default;
```

Comment:
Can we rename these striped variables to be `table-striped-...` rather than `stripe`? Just to be consistent with what we are calling this variation